terraform/stacks/main/stack.tf

```diff
@@ -305,6 +305,15 @@ module "diego" {
   log_bucket_name = module.log_bucket.elb_bucket_name
 }
 
+module "opensearch_logs_customer" {
```

Comment:
We need to make this module have a conditional `count` based on a boolean variable named `deploy_opensearch_logs_customer` or something